R/monolixReadData.R

```diff
@@ -406,7 +406,7 @@ rxUiGet.monolixCovarianceEstimatesSA <- function(x, ...) {
 #' @export
 rxUiGet.monolixCovariance <- function(x, ...) {
   .cov <- rxUiGet.monolixCovarianceEstimatesSA(x, ...)
-  .split <- ui$getSplitMuModel
+  .split <- x$getSplitMuModel
```

Comment:
I think that this is the correct fix, but please confirm @mattfidler

Reply:
Yes this is correct @billdenney